dids/src/main/kotlin/web5/sdk/dids/methods/dht/DidDht.kt

```diff
@@ -639,6 +638,12 @@ public sealed class DidDhtApi(configuration: DidDhtConfiguration) {
       else -> throw IllegalArgumentException("Unknown key type: ${data["t"]}")
     }
 
+    val verificationMethodId = when {
+      data.containsKey("id") -> data["id"]!!
+      isIdentityKey(did, keyBytes) -> "0"
+      else -> publicKeyJwk.computeThumbprint()
```

Comment:
do we have test cases to cover all 3 conditions in here?

Reply:
probably not. if we implement the [test vectors from the spec](https://did-dht.com/#test-vectors) we will be fine...

Reply:
i think this is a sufficient short term fix, we can take test vectors in a follow up

Reply:
okey dokey wrote a new issue for this https://github.com/TBD54566975/web5-kt/issues/335

Reply:
it's still good to have that issue to make sure we implement the test vectors, but since codecov was yelling at me I added a new test case so these are all covered https://github.com/TBD54566975/web5-kt/pull/334/commits/2e29bae3bcda7336db0d7790766d07cb6c16451e